test/libsolidity/semanticTests/array/copying/copy_storage_arrays_of_function_types.sol

```diff
@@ -0,0 +1,27 @@
+contract C {
+  function() external [] s0;
+  function() external view [] s1;
+  function copyStorageArrayOfFunctionType() public {
+    s0 = s1;
```

Comment:
Can you add a test where the assignment is the other way around? That should result in an error.

Reply:
Indeed yes. It gives an error. 
I ran the command solc --ir test.sol and the error I get is 

`Type function () external[] storage ref is not implicitly convertible to expected type function () view external[] storage ref.`

Is this error expected? or do we need to make some change in this behavior?

Reply:
Yes, it is expected. You have to move the changed test into the syntaxTests directory. Inside the semanticTests, everything has to compile without error.

Reply:
Okay! I'll add a third test covering this behavior in the Syntax Tests folder.

Reply:
Also check if the test is not already there.